### PR TITLE
chore: some minor DX improvements

### PR DESCRIPTION
**Pull request by xxchan**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?
playground: don't panic when PREFIX_BIN is not set.
risedev: add command hint for user-managed fe/meta

### CODE REVIEW

**Comment by xxchan** [comment]
🐳

File: src/risedevtool/src/task/frontend_service.rs [hunk 1]
```diff
@@ -97,6 +98,13 @@ impl Task for FrontendService {
             ctx.pb.set_message("started");
         } else {
             ctx.pb.set_message("user managed");
+            writeln!(
```
Comment (BugenZhao) [inline src/risedevtool/src/task/frontend_service.rs:101]: I remember we have printed this in RiseDev's log?

Comment (xxchan) [reply src/risedevtool/src/task/frontend_service.rs:101]: Only for compute

Comment (BugenZhao) [reply src/risedevtool/src/task/frontend_service.rs:101]: Good catch. 😄

**Review by BugenZhao — APPROVED** [review]

**Pull request merged**